bulb.py

```diff
@@ -0,0 +1,64 @@
+
+import datetime
+
+
+class Bulb(object):
+
+    def __init__(self,  name=None):
+        self.name = name
+        self.not_home_wasted = []
+        self.not_present_wasted = []
+        self.sleeping_wasted = []
+        self.utilized = []
+        # 3d array matrix[home][present][sleeping]
```

Comment:
This one is bit weird, because determining if a bulb is on is a bit more complex than just saying "is present around the bulb".

Think kitchen light. Kitchen light is wasted if 
- person IS home
- (person IS at desk) OR (person IS at bed)

we discussed this in person